### PR TITLE
Bootstrap 4 Support

### PR DESCRIPTION
**Pull request by 0xTim**
This PR adds the ability for the Paginator to be configured to output Bootstrap 4 compatible HTML. It also adds in accessibility support for the HTML in both Bootstrap 3 and Bootstrap 4 and allows an Aria Label to be specified for the Paginator `<nav>` tag for Screen Readers and other assistive technologies.

No breaking changes, by default it uses Bootstrap 3 with no label, it just now adds some accessibility labels.

### CODE REVIEW

File: Sources/Paginator+Leaf.swift [hunk 1]
```diff
@@ -89,7 +96,22 @@ extension PaginatorTag {
...
         var bytes: Bytes = []
         
-        let header = "<nav class=\"paginator text-center\">\n<ul class=\"pagination\">\n".bytes
+        let navClass: String
+        let ulClass: String
+        if useBootstrap4 {
+            navClass = "paginator"
+            ulClass = "pagination justify-content-center"
+        }
+        else {
```
Comment (steffendsommer) [inline Sources/Paginator+Leaf.swift]: Would you mind aligning `if else` with repo, so this line becomes `} else {`?

**Comment by steffendsommer** [comment]
Thanks a lot @0xTim. LGTM - @BrettRToomey ?

**Comment by TimC-BBC** [comment]
That's an annoying failing test! Do `URLComponents` exist on Linux Swift? (Can't check at work) - I could port the tests to extract the path queries using that and make sure each query exists?

**Comment by 0xTim** [comment]
That's an annoying failing test on Linux! Do `URLComponents` exist on Linux Swift? (Can't check at work) - I could port the existing tests to extract the path queries and make sure they all exist rather than being brittle and breaking on different orders, which shouldn't make any difference.

**Comment by steffendsommer** [comment]
Yes I believe so (https://github.com/apple/swift-corelibs-foundation/blob/master/Docs/Status.md). I think that's a good idea 👍

**Comment by 0xTim** [comment]
Cool, I'll get that in this PR then this evening probably, be nice to have a passing build on `master`!

**Comment by BrettRToomey** [comment]
Awesome! Thanks a lot! Great PR. 🙌

**Pull request merged**